2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_setup.py:_flush():67] Current SDK version is 0.19.8
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_setup.py:_flush():67] Configure stats pid to 379938
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_setup.py:_flush():67] Loading settings from /home/inside-tech/.config/wandb/settings
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_setup.py:_flush():67] Loading settings from /home/inside-tech/Desktop/image_segmentation/wandb/settings
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/inside-tech/Desktop/image_segmentation/wandb/run-20250321_155119-q2g91uj8/logs/debug.log
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/inside-tech/Desktop/image_segmentation/wandb/run-20250321_155119-q2g91uj8/logs/debug-internal.log
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_init.py:monkeypatch_ipython():599] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x71c38b747950>
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_init.py:init():761] calling init triggers
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_init.py:init():784] starting backend
2025-03-21 15:51:19,920 INFO    MainThread:379938 [wandb_init.py:init():788] sending inform_init request
2025-03-21 15:51:19,921 INFO    MainThread:379938 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-21 15:51:19,921 INFO    MainThread:379938 [wandb_init.py:init():798] backend started and connected
2025-03-21 15:51:19,923 INFO    MainThread:379938 [wandb_run.py:_label_probe_notebook():1204] probe notebook
2025-03-21 15:51:19,923 INFO    MainThread:379938 [wandb_init.py:init():891] updated telemetry
2025-03-21 15:51:19,928 INFO    MainThread:379938 [wandb_init.py:init():915] communicating run to backend with 90.0 second timeout
2025-03-21 15:51:20,265 INFO    MainThread:379938 [wandb_init.py:init():990] starting run threads in backend
2025-03-21 15:51:20,328 INFO    MainThread:379938 [wandb_run.py:_console_start():2375] atexit reg
2025-03-21 15:51:20,328 INFO    MainThread:379938 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-03-21 15:51:20,328 INFO    MainThread:379938 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-03-21 15:51:20,328 INFO    MainThread:379938 [wandb_run.py:_redirect():2315] Redirects installed.
2025-03-21 15:51:20,329 INFO    MainThread:379938 [wandb_init.py:init():1032] run started, returning control to user process
2025-03-21 15:52:14,924 INFO    MainThread:379938 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /home/inside-tech/Desktop/image_segmentation/wandb/run-20250321_155214-rmb0oxnu/logs/debug.log
2025-03-21 15:52:14,924 INFO    MainThread:379938 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /home/inside-tech/Desktop/image_segmentation/wandb/run-20250321_155214-rmb0oxnu/logs/debug-internal.log
2025-03-21 15:52:14,924 INFO    MainThread:379938 [wandb_init.py:init():761] calling init triggers
2025-03-21 15:52:14,924 INFO    MainThread:379938 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'_wandb': {}}
2025-03-21 15:52:14,924 INFO    MainThread:379938 [wandb_init.py:init():776] wandb.init() called while a run is active
2025-03-21 16:06:40,237 INFO    MainThread:379938 [wandb_run.py:_finish():2112] finishing run vincenzo-civale-universi-degli-studi-di-firenze/CowSegmentation/q2g91uj8
2025-03-21 16:06:40,237 ERROR   MainThread:379938 [jupyter.py:save_history():450] Run pip install nbformat to save notebook history
2025-03-21 16:06:40,237 INFO    MainThread:379938 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-03-21 16:06:40,237 INFO    MainThread:379938 [wandb_init.py:_jupyter_teardown():581] cleaning up jupyter logic
2025-03-21 16:06:40,238 INFO    MainThread:379938 [wandb_run.py:_atexit_cleanup():2340] got exitcode: 0
2025-03-21 16:06:40,238 INFO    MainThread:379938 [wandb_run.py:_restore():2322] restore
2025-03-21 16:06:40,238 INFO    MainThread:379938 [wandb_run.py:_restore():2328] restore done
2025-03-21 16:06:49,161 INFO    MainThread:379938 [wandb_run.py:_footer_history_summary_info():3956] rendering history
2025-03-21 16:06:49,162 INFO    MainThread:379938 [wandb_run.py:_footer_history_summary_info():3988] rendering summary
2025-03-21 16:06:49,162 INFO    MainThread:379938 [wandb_run.py:_footer_sync_info():3917] logging synced files
